packages/cli/scripts/build.mjs

```diff
@@ -54,11 +45,11 @@ function bundleOpenApiSpecs(rootDir = ROOT_DIR, specFileName = SPEC_FILENAME) {
 	shell
 		.find(publicApiDir)
 		.reduce((acc, cur) => {
-			return cur.endsWith(specFileName) ? [...acc, path.relative('.', cur)] : acc;
+			return cur.endsWith(specFileName) ? [...acc, path.relative('./src', cur)] : acc;
 		}, [])
 		.forEach((specPath) => {
 			const distSpecPath = path.resolve(rootDir, 'dist', specPath);
-			const command = `swagger-cli bundle ${specPath} --type yaml --outfile ${distSpecPath}`;
+			const command = `npm run swagger -- bundle src/${specPath} --type yaml --outfile ${distSpecPath}`;
```

Comment:
Not sure if `/` might cause issues for Win.

Reply:
Tested on windows, and it seems to work without any issues.